### ISSUE TITLE
Remove tooltip

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3906]
### Steps to recreate
Go to [search results](https://catalog.princeton.edu/?f%5Bformat%5D%5B%5D=Book) 

- Use the mouse and hover over the author name.  A tooltip is triggered with some extra information including the words 'Search' or 'Browse' accordingly.
- Use the keyboard and tabkey on the author name. The tooltip is not triggered.
- Use the screenreader. The user will get the same information with the keyboard user but not with the sighted user. 
- Use the screenreader and point the mouse on the author browse. The screenreader will prompt the use to listen to a description. 

### Issues
[Understanding SC 2.1.1](https://www.w3.org/WAI/WCAG22/Understanding/keyboard)

Pointer actions should have a keyboard equivalent 
The tooltip is adding an aria-describedby which is useful for a screenreader user but when they point the mouse on the field. This is not useful for a screenreader user. 

### WCAG Conformance level (A, AA, AAA)
A

### User impact
All users dont have the same experinece and do not get the same information.

### Recommended fix
Remove the data tooltip and title

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
